game.js

```diff
@@ -9,6 +9,8 @@ const GameStates = {
 	SOLVED: 'Puzzle solved',
 }
 
+const targetScore = 10000;
```

Comment:
I added an input to configure the target score in the game. I didn't know if you want a drop down menu that picks options for scores or just buttons with scores on them. I set a limit of 1 million because it seems  too crazy that you could just input an extremely large score.  I think currently when you complete a level it removes that from the list, so you would complete the game long before you could reach a million points. What would you suggest is a reasonable cap on the points, or should you just be able to enter as large of a number as possible?